core/src/main/java/com/orientechnologies/orient/core/storage/cache/local/OWOWCache.java

```diff
@@ -1693,9 +1689,10 @@ private void openFile(final OFileClassic fileClassic) {
 
   }
 
-  private void createFile(final OFileClassic fileClassic) throws IOException {
+  private void createFile(String fileName, int intId, final OFileClassic fileClassic) throws IOException {
     if (!fileClassic.exists()) {
       fileClassic.create();
+      writeNameIdEntry(new NameFileIdEntry(fileName, intId, fileClassic.getName()), false);
```

Comment:
Not bad but could you put this line below, so if we register file in write cache it will be already in the consistent state if the crash happens.